### ISSUE TITLE
[Pf2e] Effect Icons Off Screen

### ISSUE THREAD

**Issue by abstract** [LichFactory-Games/TotM-Manager#7]
I'm having an issue in Pathfinder 2e when using this module. The images below show exactly what the issue is. The first image shows the buff icons/text going off the top of the screen with TotM-Manager active and the second image shows what it should look like with TotM-Manager deactivated. 

Both images show that TotM-Manager is the only difference between the displays.

With TotM-Manager active
![image](https://github.com/user-attachments/assets/e4e8843f-ab8f-444b-94dd-306e75b68607)

With TotM-Manager deactivated
![image](https://github.com/user-attachments/assets/801db93f-1fc0-4e35-8f0a-b05864d59de1)

**Comment by mclearc** [comment]
Can you tell me what versions you have of foundry, pathfinder 2, and  TotMM plus its dependencies?

**Comment by abstract** [comment]
Foundry Virtual Tabletop v12.330
Pathfinder Second Edition v6.2.1
TotMM v0.8.5
libWrapper v1.12.15.0
Tagger v1.5.1
Token Magic FX v0.6.8

**Comment by mclearc** [comment]
Thanks -- I'll look into this tonight if I can. v12 has been causing more issues in various game systems than I expected.

**Comment by abstract** [comment]
I believe I've found the issue. I figured it was a CSS conflict, so I compared TotMM's styles to Pathfinder Second Edition's and I found a match. I deleted the conflicting style locally, and it fixed the issue on my machine.

The CSS conflict is the class `.effect-item` which is present in both. 

In your CSS, it is on line 328 here:
https://github.com/LichFactory-Games/TotM-Manager/blob/main/styles/totmm.css

```css
.effect-item {
    display: flex;
    align-items: center;
    padding: 5px;
}
```

In the PF2e CSS, it is in _effects-panel.scss line 7 here:
https://github.com/search?q=repo%3Afoundryvtt%2Fpf2e+effect-item&type=code

```css
    .effect-item {
        display: flex;
        justify-content: end;
        height: 52px;
```

You will just need to change the name of your CSS to something else.
